galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/SetupEcosystem.java

```diff
@@ -11,87 +11,73 @@
 import org.apache.commons.logging.LogFactory;
 import org.osgi.service.component.annotations.Component;
 
+import dev.galasa.framework.spi.ConfigurationPropertyStoreException;
 import dev.galasa.framework.spi.FrameworkException;
 import dev.galasa.framework.spi.IConfigurationPropertyStoreService;
 import dev.galasa.framework.spi.IFramework;
 
 @Component(service = { SetupEcosystem.class })
 public class SetupEcosystem {
-    
+
     private Log logger = LogFactory.getLog(this.getClass());
 
     private static final String DSS_ENV_VAR   = "GALASA_DYNAMICSTATUS_STORE";
     private static final String RAS_ENV_VAR   = "GALASA_RESULTARCHIVE_STORE";
     private static final String CREDS_ENV_VAR = "GALASA_CREDENTIALS_STORE";
-    
+    private static final String AUTH_ENV_VAR  = "GALASA_AUTH_STORE";
```

Comment:
What is this used for ?
Given we don't support Docker deployments now, do we need this ?

Reply:
This is invoked when running the boot jar with the `--setupeco` flag, and is used in the Helm chart to set the `framework.xxx.store` properties into the CPS. Would be great to remove if it's not needed, but that would likely need to be a separate story.